uwblueprint/jump-math
# (3) fix: clean up more modal logic

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
- Fix unneeded border radius on modal error message
- Update `AddOrEditStudentModal`, `DeleteStudentModal`, and `DistributeAssessmentModal` to use new props
- Remove unnecessary toasts from assessment summary page
- Update student submit button to have error handling
- Add prop validation in Typescript to `ActionButton` (temporarily disabled until all modals are updated)



## Steps to test
1. View the associated pages (student drilldown page, distribute assessment page, and student submission page)
2. Verify that the modals on those pages work as expected.



## What should reviewers focus on?
* 


## Checklist
